tests/integration/test_system_containers_mount.sh

```diff
@@ -65,7 +65,7 @@ OUTPUT=$(! ${ATOMIC} mount --live ${NAME} ${WORK_DIR}/mount 2>&1)
 grep "do not support --live" <<< $OUTPUT
 
 
-# 4. Check that --shared works
-${ATOMIC} mount --shared ${NAME} ${WORK_DIR}/mount
+# 4. Check that --shared works and that 'http:' is dropped
+${ATOMIC} mount --shared http:${NAME} ${WORK_DIR}/mount
```

Comment:
the OSTree storage doesn't use a different selinux mount label since system containers do not require a new file system when used.  We do not create any new mount point, we only checkout the files to the file system (`/var/lib/containers/atomic/$ID.0`) so they all have the same mount label.  Since it is `shared` in any case, the `--shared` option is simply ignored.